spec/system/targets/show_spec.rb

```diff
@@ -64,7 +64,7 @@
     # Header should have the title and the status of the current status of the target.
     within('.course-overlay__header-title-card') do
       expect(page).to have_content(target_l1.title)
-      expect(page).to have_content('Pending')
+      expect(page).not_to have_content('Pending')
```

Comment:
Similar to what I wrote before, I don't think an absence check makes sense here.

There's more of the same in this file.